sriovnet_switchdev.go

```diff
@@ -287,6 +293,8 @@ func GetRepresentorMacAddress(netdev string) (net.HardwareAddr, error) {
 	}
 	uplinkPhysPortName := "p" + portNum[1]
 	// Find uplink netdev for that port
+	// Note(adrianc): As we support only Smart-NICs ATM we do not need to deal with netdevs from different
+	// eswitch (i.e different switch IDs).
```

Comment:
I don't understand this comment. multi smart-nic can have different switch id so I don't understand the comment

Reply:
`GetRepresentorMacAddress()` is supported only for Smart-NICs ATM (as stated in the method description). that is, it is expected to be invoked only within a Smart-NIC embedded ARM cores.
In this case we only have a single eswitch.

if we were to extend support for this method to general hosts (i.e non smart-NIC case) then switch ID would matter.

as a note for the latter, a host with multiple smart-NICs will not have the representors on the host.